src/solver/opamActionGraph.ml

```diff
@@ -71,12 +75,14 @@ module MakeAction (P: GenericPackage) : ACTION with type package = P.t
   type t = package action
 
   let compare t1 t2 =
-    (* `Install > `Build > `Upgrade > `Reinstall > `Downgrade > `Remove *)
+    (* XXX is this the right ordering for Fetch? *)
```

Comment:
Wait, don't merge just yet :-).
I just remembered there's this TODO remaining: is the ordering for `Fetch` correct (on the line below)?

Reply:
I don't think the ordering matters here, we just need it to be total.